### DISCUSSION TITLE
Handle order updates in backtesting

### DISCUSSION THREAD

**Discussion by ruslangm** [gbeced/basana#46] [Q&A] [answered]
For a backtest, is there a way to subscribe to order update events from the exchange, similar to the user data stream in Binance? From code it seems there is no way. Or is it sufficient to just call `exchange.get_order_info` on each bar event, since orders can only be filled after processing a new bar?

**Comment by cmzxo** [comment]
I think you can try to create a `EventSource` for order updates in `backtesting_exchange.Exchange`.

**Reply by gbeced** [reply, marked as answer]
Right now there is no way to get order updates using Binance or the backtesting exchange. Calling get_order_info will work, but let me see if I can add support for both features in the next version.

**Reply by gbeced** [reply]
The latest version, 1.7, includes both features.
